eth2near/eth2near-block-relay-rs/src/lib.rs

```diff
@@ -9,3 +9,4 @@ pub mod hand_made_finality_light_client_update;
 pub mod init_contract;
 pub mod logger;
 pub mod test_utils;
+pub mod relay_errors;
```

Comment:
```suggestion
pub mod relay_errors;

```

Reply:
yes, just a minor fix which could be done by clicking on `Commit suggestion`

Reply:
It's already outdated, "Commit suggestion" won't work now.

Reply:
Fixed in 6be0fd32e0727b83de330d37f8db87fd8673c0be